### DISCUSSION TITLE
Monin-Obukhov wind in relatively small domain

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#13013] [Q&A] [answered]
Hi!

Currently, I am looking into the effect of wind on the mass loss of a partially burning tree using an M-O wind profile. The wind has `DIRECTION = 270.` and is pointed in the $+x$-direction. I observed that for a domain of $2.4 \times 2.4 \times 14.4 \ \mathrm{m^3}$, the tree ($D = 1.19 \ \mathrm{m}$, $H = 3.96 \ \mathrm{m}$) lost $\sim 13 $% of its mass. 

When I extend the domain in the $+x$-direction by $10 \ \mathrm{m}$, the $\sim 13 $% mass loss becomes $\sim 19 $%. I wonder if this could be the result of using an M-O profile in a relatively small domain.

If there is no simple answer, I will create two cases and submit them to the issue tracker.

**Comment by drjfloyd** [comment]
A 2.4 m long domain for a 1.2 m diameter tree with wind seems small. I suspect you might see similar behavior with any wind profile.

**Reply by r-broek** [reply]
Thank you for your response. Would you mind elaborating on what can go wrong if the domain is too small and how I can check if this is indeed happening?

**Reply by drjfloyd** [reply, marked as answer]
If you increase the domain size and your result changes significantly, that is a pretty good indicator that your initial domain was too small.

A small domain can allow unburned fuel to escape reducing the heat feedback to the tree. Having domain boundaries too close to regions of interest can perturb the local flow field changing results.

**Reply by r-broek** [reply]
Oh, I see! Thanks a lot for the clear explanation.
